test/runner/browserstack/api.js

```diff
@@ -118,6 +118,14 @@ export async function getBrowsers( flat ) {
 	return browsers.sort( sortBrowsers );
 }
 
+function matchVersion( browserVersion, version ) {
+	if ( !version ) {
+		return false;
+	}
+	const regex = new RegExp( `^${ version.replace( /\./g, "\\." ) }\\b`, "i" );
+	return regex.test( browserVersion );
```

Comment:
Wouldn't it be enough to return `browserVersion.startsWith( version )`? What do we gain by the regex here?

Reply:
Notice the `\b`.